platform/darwin/src/MGLTileSet.h

```diff
@@ -19,6 +19,27 @@ typedef NS_ENUM(NSUInteger, MGLTileSetScheme) {
 
 /**
  An `NSArray` of `NSString` objects that represent the tile templates.
+ 
+ Tile URL template strings are a loose open standard for referring to custom
+ locations from which to download tiles, whether vector or raster.
+
+ Currently the tokens `x`, `y`, and `z` are supported for tile locations
```

Comment:
Also [`quadkey`, `bbox-epsg-3857`, and `prefix`](https://github.com/mapbox/mapbox-gl-native/blob/dbb520662d0145a0219e319a67bbe81326c144e3/src/mbgl/storage/resource.cpp#L105-L113). (`bbox-epsg-3857` is used for our [WMS support](https://www.mapbox.com/mapbox-gl-js/example/wms/).) Each of these tokens requires some explanation, so a bulleted list would be justified here.